sdk/scheduler/src/main/java/com/mesosphere/sdk/state/GoalStateOverride.java

```diff
@@ -20,6 +21,15 @@
     /** The definition of the "STOPPED" override state, where commands are replaced with sleep()s.*/
     STOPPED("STOPPED", "STOPPING");
```

Comment:
Is it in the scope of this PR to rename these to `PAUSE*`?

Reply:
no